src/BenchmarkDotNet/Detectors/Cpu/HardwareIntrinsics.cs

```diff
@@ -55,32 +57,32 @@ static IEnumerable<string> GetCurrentProcessInstructionSets(Platform platform)
                 {
                     case Platform.X86:
                     case Platform.X64:
-
-                        if (IsX86Avx512FSupported) yield return GetShortAvx512Representation();
-                        else if (IsX86Avx2Supported) yield return "AVX2";
-                        else if (IsX86AvxSupported) yield return "AVX";
-                        else if (IsX86Sse42Supported) yield return "SSE4.2";
-                        else if (IsX86Sse41Supported) yield return "SSE4.1";
-                        else if (IsX86Ssse3Supported) yield return "SSSE3";
-                        else if (IsX86Sse3Supported) yield return "SSE3";
-                        else if (IsX86Sse2Supported) yield return "SSE2";
-                        else if (IsX86SseSupported) yield return "SSE";
-                        else if (IsX86BaseSupported) yield return "X86Base";
-
-                        if (IsX86AesSupported) yield return "AES";
-                        if (IsX86Bmi1Supported) yield return "BMI1";
-                        if (IsX86Bmi2Supported) yield return "BMI2";
-                        if (IsX86FmaSupported) yield return "FMA";
-                        if (IsX86LzcntSupported) yield return "LZCNT";
-                        if (IsX86PclmulqdqSupported) yield return "PCLMUL";
-                        if (IsX86PopcntSupported) yield return "POPCNT";
+                    {
+                        if (IsX86Avx10v2Supported) yield return "AVX10v2";
```

Comment:
This prints the full set of ISAs supported based on the way we group them in RyuJIT